optuna/integration/pytorch_lightning.py

```diff
@@ -29,18 +31,21 @@ class PyTorchLightningPruningCallback(EarlyStopping):
             ``pytorch_lightning.LightningModule.training_step`` or
             ``pytorch_lightning.LightningModule.validation_epoch_end`` and the names thus depend on
             how this dictionary is formatted.
+        **kwargs: Additional kwargs for ``pytorch_lightning.callbacks.EarlyStopping``
```

Comment:
IMO, this `**kwargs` is ambiguous. In Optuna, we have no this type of arguments of classes. How about adding each argument of the superclass if you need it. Which arguments do you want to add?